src/injected/visualization/failure-instance-formatter.ts

```diff
@@ -1,10 +1,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { DialogRenderer } from '../dialog-renderer';
 import { AxeResultsWithFrameLevel, IAssessmentVisualizationInstance } from '../frameCommunicators/html-element-axe-results-helper';
 import { Formatter } from './formatter';
 
+type failureBoxConfigType = {
+    background: string;
+    fontColor: string;
+    text: string;
+    boxWidth: string;
+};
```

Comment:
formatter.ts already defines a BoxConfig type identical to this; why not reuse that?

Reply:
Done. Missed that.